### ISSUE TITLE
Add GLEAM images

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-sky#65]
I just now saw the GLEAM survey images here:
http://www.nature.com/news/gleam-highres-composite-jpg-7.41222?article=1.21156

Looking here, it seems that they have nice images of some supernova remnants and other Galactic emission components (scroll down to the GLEAM wavelength):
http://gleamoscope.icrar.org/gleamoscope/trunk/src/

There's descriptions here:
http://www.mwatelescope.org/science/gleam-survey
https://arxiv.org/pdf/1505.06041v1.pdf
but I didn't see FITS images or a HIPS version here yet:
http://aladin.u-strasbg.fr/hips/list

When a HIPS version is available, we should add it to gamma-sky.net

**Comment by tboch** [comment]
@cdeil : it seems that ICRAR already produced a HiPS:
http://ict.icrar.org/store/gleam/gleam_color/

**Comment by cdeil** [comment]
Indeed, http://ict.icrar.org/store/gleam/gleam_color/ is very nice.

We should add it to gamma-sky.net.

Their server seems to be very slow.

Maybe we should put a copy here, which is being served very fast:
https://github.com/cdeil/hips-data

@tboch - is there a command to "clone" or "download" a HiPS?

**Comment by tboch** [comment]
Yes, Hipsgen has a dedicated command `-clone mirror` for this.
Example:

`java -jar Hipsgen.jar  in="http://ict.icrar.org/store/gleam/gleam_color/" out=my_local_path -clone mirror`

**Comment by cdeil** [comment]
@tboch - Thanks for the tip with `-clone mirror`

Looks like the option is not documented:
https://gist.github.com/cdeil/0a920bdae417e92cfe33a47790000cec

It's a few GB and takes an estimated 3 hours to clone, which is not really what I wanted.

Is there any way to only clone order 3 and 4, but not order 5?
Or restrict the region to only download the tiles for the Galactic plane?

I think HiPS cloning might be something worth adding to the Python package.
Made an issue here to discuss: https://github.com/hipspy/hips/issues/122

**Comment by cdeil** [comment]
I made a clone and put it here: https://github.com/cdeil/hips-gleam-data

I did remove `Norder5`, because looking visually, it didn't contain extra info compared to `Norder4`:
```
$ du -hs Norder*
237M	Norder3
814M	Norder4
```
`Norder3` contains most info, but is visibly lower resolution, i.e. point sources are just spread across a few pixels.

**Comment by tboch** [comment]
you can use the option `hips_order=...` to restrict the maximum order you want to download.
